src/iosource/Packet.cc

```diff
@@ -613,6 +98,7 @@ RecordValPtr Packet::ToRawPktHdrVal() const
 	else if ( l3_proto == L3_ARP )
 		l3 = BifEnum::L3_ARP;
 
+	// TODO: Get rid of hardcoded l3 protocols.
```

Comment:
This is part of the discussion on how to manage meta data. It will be addressed once we move from the hardcoded fields in Packet to a more generalizable approach.